app/views/admin/user.scala.html

```diff
@@ -42,6 +42,7 @@ <h1>General Info and Stats</h1>
                         <th class="col-md-2">Mission Count</th>
                         <th class="col-md-2">Audited Street Count</th>
                         <th class="col-md-2">Label Count</th>
+                        <th class="col-md-2">Total Distance Audited</th>
```

Comment:
So this setup with the "col-md-2" and "col-lg-12", we're using [bootstrap's grid system](https://getbootstrap.com/docs/4.0/layout/grid/), which has 12 "columns". The "col-md-2" means that it takes up 2 of the 12 "columns". Adding a 7th actual column (that takes up 2 "columns") brings us to 14 total, which can mess with the look of the layout a bit.

I think for this situation, it might be best to just remove the "-2" from everything. That way, bootstrap will just set the widths of the different columns on it's own, and it will probably look better here. Sometimes we want more control than that, but I don't think we need it here.